osxcollector/output_filters/alexa/lookup_rankings.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# AlexaRankingFilter uses the AWIS API to lookup the Alexa traffic rankings of domains found in 'osxcollector_domains'. This information is consolidated and added to the 'osxcollector_alexa_rank' key.
+#
+from threat_intel.alexa import AlexaRankingsApi
+
+from osxcollector.output_filters.base_filters.output_filter import run_filter_main
+from osxcollector.output_filters.base_filters.threat_feed import ThreatFeedFilter
+from osxcollector.output_filters.util.config import config_get_deep
+
+
+class LookupRankingsFilter(ThreatFeedFilter):
+
+    """A class to lookup traffic rankings using AWIS API."""
+
+    def __init__(self, lookup_when=None, **kwargs):
+        super(LookupRankingsFilter, self).__init__('osxcollector_domains', 'osxcollector_alexa_rank', lookup_when=lookup_when, name_of_api_key=None, **kwargs)
```

Comment:
It would be good to keep the line length <80 characters for better readability.

We should also set up commit hook to check for that.

Reply:
I will update the new files to conform to such PEP 8 guidelines. Also, yes, we really should. I will create an issue and pull request for that too.